misaligned.py

```diff
@@ -4,9 +4,23 @@ def print_color_map():
     minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
     for i, major in enumerate(major_colors):
         for j, minor in enumerate(minor_colors):
+            assert(get_pair_number_from_color(major, minor) == i * 5 + j)

```

Comment:
Even after you correct the behavior of this function (due to which asserts are failing), the print will still be wrong?
Be careful about difference between what you are testing and what the user sees